src/core/library.c

```diff
@@ -1040,13 +1045,9 @@ QuicLibrarySetGlobalParam(
             CXPLAT_DBG_ASSERT(MsQuicLib.Partitions != NULL);
             CXPLAT_DBG_ASSERT(MsQuicLib.Datapath != NULL);
 
-            if (MsQuicLib.ExecutionConfig == NULL) {
-                Status = QUIC_STATUS_INVALID_STATE;
-            } else {
-                MsQuicLib.ExecutionConfig->PollingIdleTimeoutUs = Config->PollingIdleTimeoutUs;
-                CxPlatDataPathUpdateConfig(MsQuicLib.Datapath, MsQuicLib.ExecutionConfig);
-                Status = QUIC_STATUS_SUCCESS;
-            }
+            CxPlatDataPathUpdatePollingIdleTimeout(
```

Comment:
Shouldn't PollingIdleTimeoutUs still also be updated in the gobal execution config for consistency?